scripts/src/workflowtesting/checkprforci.py

```diff
@@ -76,14 +78,17 @@ def main():
             print(f"::set-output name=run-tests::true")
         else:
             print(f"[INFO] User not authorized for manual invocation - do not run tests.")
+            sys.exit(1)
```

Comment:
One way you can do this better is to raise an exception the will fail and print the message